opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/relevance/RelevanceQuery.java

```diff
@@ -81,4 +102,5 @@ public QueryBuilder build(FunctionExpression func) {
   protected interface QueryBuilderStep<T extends QueryBuilder> extends
       BiFunction<T, ExprValue, T> {
   }
+
```

Comment:
Formatting change not needed, and really should only be included in a review when appropriate.

Reply:
Fixed in 7add05fe7c1a7fcef259e1e10656f3c09e50bbc8